typings/axe-core/axe-core-tests.ts

```diff
@@ -1,7 +1,7 @@
 import * as axe from '../../axe';
 
 var context: any = document;
-var $fixture: any = {};
+var $fixture = document.querySelectorAll('div');
```

Comment:
I'm a little surprised this works, since QSA returns a NodeList rather than an array. Should we explicitly mention NodeList?

Reply:
Looked into this. It looks like it doesn't matter what exactly is passed in for the include/exclude object so long as it is array-like. Once we get the context object we immediately call [parseSelectorArray](https://github.com/dequelabs/axe-core/blob/develop/lib/core/base/context.js#L248), which loops over the context include/object object using a [for loop that looks at the length of the object](https://github.com/dequelabs/axe-core/blob/develop/lib/core/base/context.js#L133).

I'm OK if we don't mention this particular case for now. We should loop back to this and determine if it's something we explicitly want to support, and if so add an official test to our code to ensure it continues to work.

Reply:
In that case, wouldn't `ArrayLike<Node>` be the best option?

Reply:
If we decide to officially support it then ya, that should be the correct type. For now I'm going to change the ts test to not use querySelectorAll and will update it back in another pr if we decide to support it.